src/components/core/index.ts

```diff
@@ -2,3 +2,4 @@ export { default as Button } from './button/button';
 export { default as Text } from './text/text';
 export { default as Checkbox } from './checkbox/checkbox';
 export { default as Tabs } from './tabs/tabs';
+export { default as DesktopWizard } from './wizard/desktop-wizard';
```

Comment:
Can we also export types used in `DesktopWizard`? It will be helpful if we can get `MainComponentProps`, `StepData` types.